en/case-studies/vanilla-survey.rst

```diff
@@ -0,0 +1,73 @@
+.. |br| raw:: html
+
+  <br/>
+
+Vanilla Surveys using QField
+============================
+
+By Ahi Saipaia (1), Leody Vainikolo (1), and John Duncan (2 and 3) |br|
+1 Ministry of Agriculture, Forestry, Food, and Fisheries, Government of Tonga |br|
+2 UWA School of Agriculture and Environment, The University of Western Australia |br|
+3 Geography and Planning, The University of Western Australia |br|
```

Comment:
```suggestion
| By Ahi Saipaia (1), Leody Vainikolo (1), and John Duncan (2 and 3)
| 1 Ministry of Agriculture, Forestry, Food, and Fisheries, Government of Tonga
| 2 UWA School of Agriculture and Environment, The University of Western Australia
| 3 Geography and Planning, The University of Western Australia
```

Does that work?

Reply:
yes, sorry I didn't understand your comment